src/i18n/locales/en/translation.json

```diff
@@ -22,12 +22,12 @@
         }
       },
       "set_password": {
-        "title": "Protect your wallet",
-        "description": "Your wallet is securely encrypted with a password and needs to be unlocked before usage.",
+        "title": "Set your an unlock password",
+        "description": "Your Lightning account data is securely encrypted with an unlock password. Do not forget this password you need it to unlock the Alby Extension",
```

Comment:
Better with a new sentence in between:

"Your Lightning account data is securely encrypted with an unlock password. Do not forget this password! You need it to unlock the Alby Extension (in this browser)"